src/modelmanager.hpp

```diff
@@ -202,7 +240,7 @@ class ModelManager {
         return models;
     }
 
-    void startSequenceCleaner();
+    void startCleaner();
```

Comment:
Could we have this inside protected/private?

Reply:
Doing so will break one test. This test checks whether deinitialize is called when shared_ptr.count == 1, which means only cleaner owns the resource.